### ISSUE TITLE
[Feature Request] Закладки на сообщения в чатах

### ISSUE THREAD

**Issue by Namsaraev** [kotatogram/kotatogram-desktop#161]
Нужна функция "Закладка" на сообщении в чатах. 
Ставится метка на одном сообщении чтобы в дальнейшем к ней перейти.
Функция полезна в больших переписках, для постепенного прочтения и возврата к закладке для продолжения чтения с помеченного места. Так же для отметки в переписке важного сообщения. В любом чате можно отметить любое сообщение данной меткой "Закладка", а через меню данного чата "перейти к Закладке", если закладка существует в данном чате.
Данная функция реализована в клиенте Telegraph.

**Comment by ilya-fedin** [comment]
> Данная функция реализована в клиенте Telegraph.

Это десктоп-клиент? Если нет, то пометка бесполезна, из могильного клиента код не взять. :(

**Comment by Namsaraev** [comment]
> > Данная функция реализована в клиенте Telegraph.
> 
> Это десктоп-клиент? Если нет, то пометка бесполезна, из могильного клиента код не взять. :(

Telegraph это клиент под андроид только, но я написал про него чтобы было понятно как работает эта функция там. Если от туда ее не взять , то добавить значить не возможно?

**Comment by ilya-fedin** [comment]
Это просто значит, что придется смотреть и имплементить с нуля, что означает, что это может занять значительное время, если кому-нибудь вообще будет это интересно.

**Comment by Namsaraev** [comment]
> Это просто значит, что придется смотреть и имплементить с нуля, что означает, что это может занять значительное время, если кому-нибудь вообще будет это интересно.

Понятно, но может кто отпишет тут, где есть еще в других клиентах подобная функция. Просто я другие клиенты не пользовал. По поводу интереса, думаю много народа хотели бы, но не все конечно знают куда заходить и голосовать за это.

**Comment by ilya-fedin** [comment]
>  По поводу интереса, думаю много народа хотели бы, но не все конечно знают куда заходить и голосовать за это.

Нет, я имею в виду интересна того, кто сможет запилить. Клиент пилится на добровольной основе и все пилят то, что хотят.

**Comment by Namsaraev** [comment]
> > По поводу интереса, думаю много народа хотели бы, но не все конечно знают куда заходить и голосовать за это.
> 
> Нет, я имею в виду интересна того, кто сможет запилить. Клиент пилится на добровольной основе и все пилят то, что хотят.

Понятно, спасибо за разъяснение.
